CHANGELOG.md

```diff
@@ -18,6 +18,16 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [0.24.0] - 2024-12-04
 
+### Release highlights
+
+* New [`jj absorb`](https://martinvonz.github.io/jj/latest/cli-reference/#jj-absorb) command automatically moves changes from the current commit into relevant ancestor commits.
```

Comment:
Should we say "squashes" instead of "moves" to hopefully make it clearer to user who are familiar with the `jj squash` command?

Reply:
Sure, but FYI I copied the description from the `jj absorb` command itself, which says "moves" rather than "squashes". (I think the `jj absorb` docs should be improved separately because it doesn't say anything like that it tries to move lines/hunks to the most recent commit that touched the same lines/hunks, so it's a bit mysterious.)

Reply:
Changed "moves" -> "squashes".